cloudinit/sources/DataSourceEc2.py

```diff
@@ -31,6 +31,9 @@
 API_TOKEN_ROUTE = 'latest/api/token'
 API_TOKEN_DISABLED = '_ec2_disable_api_token'
 AWS_TOKEN_TTL_SECONDS = '21600'
+AWS_TOKEN_PUT_HEADER = 'X-aws-ec2-metadata-token'
+AWS_TOKEN_REQ_HEADER = AWS_TOKEN_PUT_HEADER + '-ttl-seconds'
+AWS_TOKEN_REDACT = [AWS_TOKEN_PUT_HEADER, AWS_TOKEN_REQ_HEADER]
```

Comment:
I'd suggest we shouldn't redact AWS_TOKEN_REQ_HEADER as this request is just the number of seconds that cloud-init asks for a token  to be valid. It is not a 'secret' and would be helpful when debugging in the future if this value did get changed or if tokens did not live as long as they were supposed to.

Reply:
I don't think so.  In the case that we lowered the value to say something like 1; the worst case is that we attempt to fetch a new token more often;  The only failure paths we have are attempting to obtain a token in the first place; and if that fails we revert to V1.  I'd rather just not log any of those values, if that makes sense.

Reply:
works for me. Just recognizing we now have to look back at cloud-init/sources/DataSourceEc2.py if we do reduce that ttl to see what that version of cloud-init is requesting. I won't make this a blocking point though.